src/screens/Accounts/Accounts.js

```diff
@@ -81,34 +82,30 @@ type ListItem = {|
 
 type Props = {|
   blockchainNetworks: BlockchainNetwork[],
-  baseFiatCurrency: ?string,
   accounts: Accounts,
   switchAccount: (accountId: string) => void,
-  balances: BalancesStore,
-  rates: Rates,
-  fetchAllAccountsBalances: () => void,
+  fetchAllAccountsTotals: () => void,
   keyBasedWalletHasPositiveBalance: boolean,
+  accountsTotals: AccountsTotalBalances,
 |};
 
 const AccountsScreen = ({
-  fetchAllAccountsBalances,
+  fetchAllAccountsTotals,
   accounts,
   switchAccount,
   blockchainNetworks,
-  balances,
-  rates,
-  baseFiatCurrency,
   keyBasedWalletHasPositiveBalance,
+  accountsTotals,
 }: Props) => {
   const navigation = useNavigation();
   const theme = useTheme();
+  const fiatCurrency = useFiatCurrency();
 
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  useEffect(() => { fetchAllAccountsBalances(); }, []);
+  useEffect(() => { fetchAllAccountsTotals(); }, []);
```

Comment:
```suggestion
  useEffect(() => fetchAllAccountsTotals(), []);
```

Maybe?

Reply:
No, `useEffect` return is reserved for `cleanup` before re-render happens to unsubscribe from prev. instances and etc. (ref – https://reactjs.org/docs/hooks-effect.html#example-using-hooks-1).